pages/delivery.md

```diff
@@ -0,0 +1,54 @@
+
+## Agile
+Agile can mean a lot of things. For us, it means building something small enough to address a well-defined need in a short timespan. It also means allowing the team to adjust their focus if user or agency feedback suggests we're on the wrong track. 
+
+Deadlines can change. Needs can change. We make our teams flexible enough to always deliver something high quality and high value. Developing, standing up, and sharing with users a minimally viable product — one that isn’t fully functional — is a key component of what we do. This allows us to gather feedback and make edits before a product is finalized.
+
+## Design centric and data driven
+User experience and usability research is embedded into our projects from the start. We want to build the right thing, not just anything. Accomplishing this means speaking to users regularly and having them test our work as often as possible. 
+
+Our developers, designers, and project managers work side-by-side with researchers to understand the impact of our design decisions. Our product teams care about usability and are often found alongside user-experience researchers testing versions of our products to understand how people interact with them.
+
+##Contact us
+If you work at an agency and are interested in partnering with 18F, send us an email at <a href="mailto:inquiries@18f.gsa.gov" itemprop="email"><i title= "email 18F"></i> inquiries@18f.gsa.gov</a>.
+
+##Learn more about the work we do
+- [Taking the pulse of the federal government’s web presence](https://18f.gsa.gov/2015/06/02/taking-the-pulse-of-the-federal-governments-web-presence/)
+- [Meet MyUSA: Your one account for government](https://18f.gsa.gov/2015/05/18/myusa/)
+- [Announcing the CALC tool: making pricing research easier in federal procurement](https://18f.gsa.gov/2015/05/12/announcing-the-calc-tool/)
+- [Flexibility when releasing a new product: Peace Corps’ new donations platform](https://18f.gsa.gov/2015/04/09/flexibility-when-releasing-a-new-product-peace-corps-new-donation-platform/)
+- [18F by the numbers](https://18f.gsa.gov/2015/03/19/18f-by-the-numbers/)
+- [How we built analytics.usa.gov](https://18f.gsa.gov/2015/03/19/how-we-built-analytics-usa-gov/)
+- [A new look at the Freedom of Information Act](https://18f.gsa.gov/2014/09/04/a-new-look-at-the-freedom-of-information-act/)
+- [Creating an OpenFEC](https://18f.gsa.gov/2014/08/21/creating-an-open-fec/)
+- [A few notes on NotAlone.gov](https://18f.gsa.gov/2014/05/09/a-few-notes-on-notalone-gov/)
+
+
+
+<!-- Obfuscate our email -->
+<div>
+  <script>
+    var parts = ["18FC", "@", "gsa", ".gov"];
```

Comment:
@emileighoutlaw change the "18FC" to "inquiries" or whatever the part of the email address before the @ is supposed to be.